spin-sdk/src/modules/spinSdk.ts

```diff
@@ -2,16 +2,62 @@ import { router, routerType } from "./router"
 import { utils } from "./utils"
 
 interface SpinConfig {
+    /**
+     * Returns the value of the given config key
+     * 
+     * @param arg0 - The name of the key
+     * @returns - A value string for the given key
+     */
     get(arg0: string): string
 }
 
 interface KvStore {
+    /**
+     * Delete the key
+     * 
+     * @param arg0 - The name of the key
```

Comment:
```suggestion
     * @param key - The name of the key
```

(looks like this might be throughout)